src/server.c

```diff
@@ -944,6 +951,68 @@ static void process_big_cut_text(struct nvnc_client* client)
 	client->cut_text.buffer = NULL;
 }
 
+static void send_extended_desktop_size(struct nvnc_client* client,
+				       enum rfb_resize_initiator initiator,
+				       enum rfb_resize_status status)
+{
+	struct rfb_server_fb_update_msg head = {
+		.type = RFB_SERVER_TO_CLIENT_FRAMEBUFFER_UPDATE,
+		.n_rects = htons(1),
+	};
+
+	struct rfb_server_fb_rect rect = {
+		.encoding = htonl(RFB_ENCODING_EXTENDEDDESKTOPSIZE),
+		.x = initiator,
```

Comment:
Don't these need to be in network order?

Reply:
That is correct, fixed now.